app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_data = {}
+
+    def wrapper(*args) -> None:
+
+        cache_data_check = args
+
+        if cache_data_check in cached_data:
+            print("Getting from cache")
+        else:
+            result = func(*args)
+            cached_data.update({cache_data_check: result})
```

Comment:
Try to write it into one line, and think about do you need an update method here, or it can be simplified